resources/views/auth/register.blade.php

```diff
@@ -55,10 +55,9 @@
                                 @endif
                             </div>
                         </div>
-
+                        <!--
                         <div class="form-group row {{ $errors->has('birth') || $errors->has('birth_year') || $errors->has('birth_month') || $errors->has('birth_day') ? ' has-error' : '' }}">
-                            <label for="birth_year" class="col-md-4 col-form-label text-md-right">Birthday　DD/MM/YYYY </label>
-
+                            <label for="birth_year" class="col-md-4 col-form-label text-md-right">Birthday</label>
                             <div class="form-group row col-md-8">
```

Comment:
this too

Reply:
Changed in line with the latest Wire Frame.